site/docs/v1/tech/apis/_webauthn-authenticate-start-response-body.adoc

```diff
@@ -0,0 +1,45 @@
+==== Response Body
+
+[.api]
+[field]#options# [type]#[Object]#::
+The WebAuthn public key credential request options. This object is passed to the WebAuthn client API to generate an assertion using an available passkey.
+
+[field]#options.allowCredentials# [type]#[Array]#::
+An array containing identifying information for passkeys that are allowed to complete this authentication ceremony, ordered from most to least preferred.
+
+[field]#options.allowCredentials``[x]``.id# [type]#[String]#::
+The credential Id generated by the authenticator as a base64url-encoded string. This value is used to locate an authenticator that can complete the ceremony.
++
+These values must be converted to JavaScript ``ArrayBuffer``s before the [field]#options# are passed to the WebAuthn client API.
+
+[field]#options.allowCredentials``[x]``.transports# [type]#[List<String>]#::
+A list of transport types supported by the authenticator that generated the passkey. This value is used as a hint to help identify authenticators containing one of the [field]#options.allowCredentials#.
```

Comment:
We did have them defined in an enum, but then Safari broke it when they added `hybrid` because the string couldn't be parsed to a known enum value.

We decided to just keep them as strings. It doesn't really matter what the value is. Maybe it would be good to document some known values in the Developer Guide?

Reply:
Makes sense. I guess if I'm using this API my first question will be 'what are the allowed values'. If they don't matter, we should say so.